bin/setup

```diff
@@ -39,7 +39,7 @@ PRISMIC_API=https://brf-energi.cdn.prismic.io/api
 MONGO_URL=mongodb://localhost:27017/brf-energi-dev
 `
 
-readFile('.env').then(content => {
+readFile('.env.client-public').then(content => {
```

Comment:
Just starting a discussion here about the name change to work as a "TODO". No need for more comment, just want something to remind me.